### ISSUE TITLE
Clarification

### ISSUE THREAD

**Issue by KatCXD** [trigaten/Learn_Prompting#489]
I'm just getting started reading through the course and went to set up a Dyno account, which requests my OpenAI API key.

I can see where to get this on openai.com but OpenAI specifically states that my API key is secret, and:
**"Do not share your API key with others, or expose it in the browser or other client-side code..."**

...so, do we just have to ignore this to use Dyno with learnprompting.org? Am I just being overly paranoid? It would be great if you could add a note about this where you first mention Dyno in the Introduction and the need to provide an OpenAI API key.

Thanks!
(Note: I also asked this in the Discord community)

**Comment by trigaten** [comment]
Hello, I plan to add a page on getting set up with Dino so it is more clear. You don't need to make a Dyno account. You just need to paste it in the box. 
> "Do not share your API key with others, or expose it in the browser or other client-side code..."
> ...so, do we just have to ignore this to use Dyno with learnprompting.org? 

Every other LLM app asks you to enter an API key, so I think they mean just be careful :)

**Comment by KatCXD** [comment]
Ah fair enough! (I did later see that it clarifies that it's saved in local storage too.)

I'm pretty much new to all this, so not that familiar with using other LLM apps...yet. :) (And I deleted my question from Discord too, just fyi...after I realised I'd posted in totally the wrong channel.) 🤦‍♀️ 

Thanks for an amazingly quick response - and for putting together an even more amazing resource for everyone out there wanting to get up to speed with prompt engineering. You are awesome. 🙇‍♀️
